docs/modules/usage/changelog.md

```diff
@@ -2,9 +2,9 @@
 sidebar_position: 8
 ---
 
-# Changelog
+# Upgrade Guide
```

Comment:
Changed this to Upgrade Guide for now until we know better what we will be doing in terms of CHANGELOG.